mono/mini/il2cpp-stubs.cpp

```diff
@@ -750,8 +748,8 @@ Il2CppMonoString* il2cpp_mono_ldstr_checked(Il2CppMonoDomain* domain, Il2CppMono
 
 Il2CppMonoObject* il2cpp_mono_runtime_try_invoke(Il2CppMonoMethod* method, void* obj, void** params, Il2CppMonoObject** exc, MonoError* error)
 {
-	IL2CPP_ASSERT(0 && "This method is not yet implemented");
-	return NULL;
+	mono_error_init(error);
+	return (Il2CppMonoObject*)il2cpp::vm::Runtime::Invoke((MethodInfo*)method, obj, params, (Il2CppException**)exc);
```

Comment:
As with `il2cpp_object_new`, we could call `il2cpp_runtime_invoke` here, but I'm unsure about what is the correct thing to do. We should at least choose the same in both cases. At the moment, the code is calling to the C++ vm layer in both cases, which is probably fine.

Reply:
Per our previous discussion, we don't make il2cpp-api calls in the stubs file, we use the C++ api.